ui/app/components/clients/history.js

```diff
@@ -89,48 +90,68 @@ export default class History extends Component {
     );
   }
 
-  get latestUpgradeData() {
-    // {id: '1.9.0', previousVersion: null, timestampInstalled: '2021-11-03T10:23:16Z'}
-    // version id is 1.9.0 or earliest upgrade post 1.9.0, timestamp is RFC3339
-    // need to return the last in the array as the data is in ascending order
-    return this.args.model.versionHistory.at(-1) || null;
+  get upgradeVersionHistory() {
+    const versionHistory = this.args.model.versionHistory;
+    if (!versionHistory || versionHistory.length === 0) {
+      return null;
+    }
+
+    // get upgrade data for initial upgrade to 1.9 and/or 1.10
+    let relevantUpgrades = [];
+    const importantUpgrades = ['1.9', '1.10'];
+    importantUpgrades.forEach((version) => {
+      let findUpgrade = versionHistory.find((versionData) => versionData.id.match(version));
+      if (findUpgrade) relevantUpgrades.push(findUpgrade);
+    });
+
```

Comment:
If neither upgrade is present, then we want just the first upgrade that happened (all upgrades will be post 1.9, because that was when the `version-history` endpoint starts)

Reply:
I like this approach, but don't we want to search the reverse of the array, e.g. from the back? Find the most recent first? Or maybe it doesn't matter.